src/LodeRunner.UI/src/components/IncompleteTestRuns/index.js

```diff
@@ -0,0 +1,49 @@
+import { A, navigate } from "hookrouter";
+import { useMemo, useContext } from "react";
+import { TestRunsContext } from "../../contexts";
+import getMMMDYYYYhmma from "../../utilities/datetime";
+import { TEST_RUN } from "../../models";
+import "./styles.css";
+
+const IncompleteTestRuns = () => {
+  const { testRuns } = useContext(TestRunsContext);
+  const incompleteTestRuns = useMemo(
+    () =>
+      testRuns.filter(
```

Comment:
Not in issue, but since there's a See All Test Runs button, would be nice to only display latest 20 or so